.circleci/config.yml

```diff
@@ -3,6 +3,19 @@ version: 2.1
 orbs:
   prodsec: snyk/prodsec-orb@1
 
+jobs:
+  security-scans:
+    resource_class: small
+    docker:
+      - image: cimg/openjdk:17.0
+    steps:
+      - checkout
+      - prodsec/security_scans:
+          mode: auto
+          open-source-scan-all-projects: false
+          open-source-additional-arguments: --maven-aggregate-project -- -pl '!tests'
```

Comment:
@bastiandoetsch I noticed also that feature and update-site modules don't contain any code. are these relevant for the scan or can they be ignored?

Reply:
They can be ignored - they only contain config files